m02_setup/04_vs_code.rst

```diff
@@ -0,0 +1,115 @@
+Set up Visual Studio Code
+=========================
+
+.. styled-topic::
+
+  Main questions:
+    How do I set up VS Code to start creating a new course in A+?
+
+  Topics:
+    In this section, we talk about:
+
+    * `VS Code`_
+    * `Install python libraries`_
+    * `Install VS Code extensions`_
+
+  Material:
+    In this chapter, you won't need extra material.
+
+  Requirements:
+    You need basic computational skills to install some software in the Linux OS.
+
+  Estimated working time:
+    From 30 min to 1 hour.
+
+
+VS Code
+-------
+
+As has been mentioned before, VS Code is an advanced IDE that will allow you to write markup documents more efficiently than using simple text editors. VS Code has a massive set of tools and extensions that will help you to avoid some of the most common mistakes during the creation of your course content.
+
+Installing VSCode is relatively straightforward. You only need to follow the instruction provided in the `VSCode <https://code.visualstudio.com/download>`_
+official website.
+
+
+Install python libraries
+------------------------
+
+At this point, you should have Python 3 already installed. However, if you do not have it installed, Download python, from the `python website <http://www.python.org/downloads>`_. 
+
+Installing python and libraries is not always an easy task. Nevertheless, we will guide you to install the python libraries without pain. First of all, you should open a new Linux terminal. Afterwards, you need to install some python libraries by typing the following commands.
+
+1. Install **Sphinx 1.6.7** 
```

Comment:
When teachers compile the RST materials with our `docker-compile.sh` script, they don't need to install Sphinx in their computers. Is this Sphinx installation in the instructions needed in order to use VS Code and to render HTML output quickly inside the editor (preview)?

Usually, pip packages are installed in a virtual environment in order to avoid breaking the system Python installation. Would it be better to use a virtualenv here too?